### DISCUSSION TITLE
Icons covering text in extensions context menu

### DISCUSSION THREAD

**Discussion by txbq** [black7375/Firefox-UI-Fix#839] [Q&A]
Hi, I've noticed that icons in the context menu for extension settings are covering the text. I'm on macOS.
![Screenshot 2023-12-12 at 8 10 03 AM](https://github.com/black7375/Firefox-UI-Fix/assets/153625745/775bbfea-36df-4e88-8a85-442c8330e889)

**Comment by black7375** [comment]
I am create to #847

I think it's a bug, and I'll look into it further over the weekend.
